src/chapters/http/web-addresses.rst

```diff
@@ -0,0 +1,123 @@
+Web Addresses
```

Comment:
This is a good section, but there aren't any Try It! or concept checks, so I'm concerned students won't really absorb these concepts. A couple of very basic, active prompts could go a long way. 

For example:
- For the URL `https://launchcode.org/lchs` what is the host and path? Is there a query string?
- For the URL `https://google.com/?q=python` what are the key/value pairs in the query string? What is different about the results returned if we change the query string to `q=html`?

Reply:
I added a Try It box and three concept check questions to address this.